### ISSUE TITLE
About the dimension of the output of the discriminator.

### ISSUE THREAD

**Issue by Simon4john** [XHUJOY/CycleGAN-tensorflow#15]
Hello!

I find the dimension of the output of the discriminator is # h4 is (32 x 32 x 1), and then the code calculate the loss ：

    # losses
    self.g_loss = self.criterionGAN(self.DA_fake, tf.ones_like(self.DA_fake)) 

I am so confused,  as I think the the dimension of the output of the discriminator should be 1.

Could you please give some hints? 

THX

**Comment by XHUJOY** [comment]
`self.criterionGAN` will use `tf.reduce_mean` to change  the dimension of loss to 1.
https://github.com/XHUJOY/CycleGAN-tensorflow/blob/master/model.py#L29
https://github.com/XHUJOY/CycleGAN-tensorflow/blob/master/module.py#L143
